docs/content/custom-replication-system.md

```diff
@@ -2,16 +2,46 @@
 
 -----
 
+## Writing a custom replication system
 
-## Custom replication systems
+### What the Unity GDK's replication system does
 
-By default in the Unity GDK, components are automatically replicated to SpatialOS whenever a component's fields are modified. However if some components require more complex replication logic, the Unity GDK supports creating custom replication systems on a per component basis by extending the `CustomSpatialOSSendSystem<T>` class, where `T` is a component.
+By default, the Unity GDK automatically replicates ECS components to SpatialOS whenever you modify an ECS component (that corresponds to a SpatialOS component)'s properties.
 
-For standard replication, each component has an internal bool named `DirtyBit`. When any property within the component is set, the dirty bit is set to `true`. The `SpatialOSSendSystem` then checks the `DirtyBit` of each component. If set to `true`, a component update is pushed and `DirtyBit` is set to `false`.
+#### For properties
 
-A custom replication system needs to handle the testing and setting of the `DirtyBit` explictly since the standard replication is not being executed.
+Each ECS component has an internal bool named `DirtyBit`. When a worker set any property of a SpatialOS component in the corresponding ECS component, the Unity GDK sets `DirtyBit` to `true`. The `SpatialOSSendSystem` then checks the `DirtyBit` of each ECS component. If set to `true`, the Unty GDK pushes a SpatialOS component update and sets `DirtyBit` to `false`.
 
-Example of a custom replication system for a `Transform` component:
+#### For events
+
+When a worker sends a SpatialOS event, the Unity GDK puts the event object into an internal buffer. When it's time to replicate a component, the GDK sends all buffered events and clears the buffer.
+
+### Writing your own replication system
+
+If some ECS components need more complex replication logic, you can create custom replication systems on a per-component basis. To do this:
+
+* Your custom replication system must extend the `CustomSpatialOSSendSystem<T>` class (where `T` is a SpatialOS component).
+
+* Deal with properties:
+
+    If you write a custom replication system that works with properties, it needs to handle the testing and setting of the `DirtyBit` explictly, because the standard replication won't happen.
```

Comment:
Move "because the standard replication won't happen" to the line about extending CustomSpatialOSSendSystem.

I.e. -  Your custom replication system must extend the `CustomSpatialOSSendSystem<T>` class (where `T` is a SpatialOS component). Note that this will disable the standard replication for `T`